lib/qesapdeployment.pm

```diff
@@ -1864,6 +1865,22 @@ sub qesap_az_clean_old_peerings {
     }
 }
 
+=head2 qesap_az_get_native_fencing_type
+
+    qesap_az_get_native_fencing_type();
+
+    Gets the native fencing type (spn/msi)
+
+=cut
+
+sub qesap_az_get_native_fencing_type {
+    my $type = get_var('AZURE_FENCE_AGENT_CONFIGURATION', get_var('QESAP_AZURE_FENCE_AGENT_CONFIGURATION', 'msi'));
```

Comment:
```suggestion
    my $type = get_var('AZURE_FENCE_AGENT_CONFIGURATION', get_var('QESAP_AZURE_FENCE_AGENT_CONFIGURATION', 'msi'));
```

Reply:
I'm confused, what's the difference

Reply:
agree, not so much, no need to change.

Reply:
Ah, you meant to change the order! But the order is the same in your suggested change

Reply:
:facepalm: